tests/test_ujson.py

```diff
@@ -600,6 +600,32 @@ def test_decode_numeric_int_exp(test_input):
     assert output == json.loads(test_input)
 
 
+@pytest.mark.parametrize(
+    "i",
+    [
+        -(10**25),  # very negative
+        -(2**64),  # too large in magnitude for a uint64
+        -(2**63) - 1,  # too small for a int64
+        2**64,  # too large for a uint64
+        10**25,  # very positive
+    ],
+)
+@pytest.mark.parametrize("mode", ["encode", "decode"])
+def test_encode_decode_big_int(i, mode):
+    # Test ints that are too large to be represented by a C integer type
+    for py in (i, [i], {"i": i}):
```

Comment:
Variable naming nit: why `py`? Can it be named more descriptively?

Reply:
To fit into the line length limits when I still had the pairs with f-strings in the loop, I think, but that's moot now. Will rename them to `pythonObject` and `jsonString`.

Reply:
Thanks, let's also use `snake_case` :)

Reply:
Right, still half-asleep. :-) (`camelCase` is what I use in my own projects.)